spyder_line_profiler/spyder/widgets.py

```diff
@@ -1,14 +1,11 @@
 # -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2021, Spyder Line Profiler contributors
 #
-# Copyright © 2011 Spyder Project Contributors
-# Licensed under the terms of the MIT License
-# (see LICENSE.txt for details)
-
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
 """
-Line Profiler widget
-
-See the official documentation of line_profiler:
-http://pythonhosted.org/line_profiler/
+Spyder Line Profiler 5 Main Widget.
```

Comment:
```suggestion
Spyder Line Profiler Main Widget.
```

No need to mention the version.